deployments/helm/nvidia-dra-driver-gpu/templates/controller.yaml

```diff
@@ -20,6 +20,7 @@ metadata:
   name: {{ include "nvidia-dra-driver-gpu.name" . }}-controller
   namespace: {{ include "nvidia-dra-driver-gpu.namespace" . }}
   labels:
+    app: {{ .Chart.Name }}
     {{- include "nvidia-dra-driver-gpu.labels" . | nindent 4 }}
```

Comment:
This part of the patch raises two interesting points:

- `include "nvidia-dra-driver-gpu.labels` already includes a label with the Helm chart name. That label's name is `app.kubernetes.io/name`.

- We hardcode `app` to be the chart name -- that is, there is no other choice. By that logic, we could remove one layer of indirection, and rename `AppLabelValue` to `ChartName` in Go code.

Reply:
Umm, yes and no. Because the CharName can be overwritten with Helm flags. What we are looking at here is a constant and persistent AppLabel that we can use to filter all our components in a cluster. Regardless of how complicated the user has installed them, a constant label gives us that single point to filter for the project components.

Reply:
> .Chart.Name cannot be overridden.

btw, ref docs: https://helm.sh/docs/chart_template_guide/builtin_objects/

And for the record, I wondered about `--generate-name` which I had used, documented here: https://helm.sh/docs/helm/helm_install/

```
--generate-name                              generate the name (and omit the NAME parameter)
```

This is _not_ the chart name (and these docs aren't great).